web/src/utils.js

```diff
@@ -60,6 +60,30 @@ function getDandisetContact(dandiset) {
   return null;
 }
 
+const basicDandisetSortingOptions = [
```

Comment:
Any reason you don't just `export const basicDandisetSortingOptions` here? Or more generally just export everything inline with where it's declared in this file?

Reply:
Things were already being exported through the object at the end of the file, so I just added to that.